content/en/intelligent_test_runner/_index.md

```diff
@@ -40,6 +40,7 @@ There are several configuration mechanisms that you can use in these scenarios t
 - You can mark certain files in your repository as [tracked files](#tracked-files), which causes all tests to run whenever these files are changed. Dockerfiles, Makefiles, dependency files, and other build configuration files are good candidates for tracked files.
 - You can mark certain tests in your source as unskippable to ensure they are always run. This is a good fit for data-driven tests or tests that interact with external systems. More information in the [setup page][3].
 - If you are authoring a risky commit and you'd like to run all tests, add `ITR:NoSkip` (case insensitive) anywhere in your Git commit message.
+- If GitHub is your source code management provider, you can label your pull requests with the `ITR:NoSkip` label (case insensitive). This label will prevent tests from being skipped in pull requests. To use this feature, you must configure the GitHub App using the [GitHub integration tile][9] with the `Software Delivery: Collect Pull Request Information` feature enabled.
```

Comment:
```suggestion
- If GitHub is your source code management provider, use the `ITR:NoSkip` label (case insensitive) to prevent Intelligent Test Runner from skipping tests in pull requests. To use this feature, configure the GitHub App using the [GitHub integration tile][9] with the `Software Delivery: Collect Pull Request Information` feature enabled.
```